src/aspire/source/simulation.py

```diff
@@ -245,6 +248,115 @@ def _apply_sim_filters(self, im, indices):
             self.filter_indices[indices],
         )
 
+    def estimate_signal_mean(self, sample_n=100):
+        """
+        Estimate the signal mean of `sample_n` projections.
+
+        :param sample_n: Number of projections used for estimate.
+        :returns: Estimated signal power.
```

Comment:
I'm confused about the terminology. Are power and mean interchangeable? I thought mean was like a normalized power maybe

Reply:
Don't over think. This was simply a typo.